assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -613,6 +613,10 @@ form {
   a, a:visited {
     color: gu-colour(news-garnett-main-1);
   }
+
+  .component-animated-dots div {
```

Comment:
any way we can avoid the nested selector?

Reply:
could `component-animated-dots` take an `appearance={light|dark}` parameter?

Reply:
and add it to storybook? 
![face-with-pleading-eyes_1f97a](https://user-images.githubusercontent.com/11539094/52433056-00839e00-2b04-11e9-8c1c-009d03e4e56f.png)

Reply:
actually we use the same technique on the dots after we contribute, and the loading screen on the speed test (I copied the former for the latter).
Perhaps I should have had an extra class on each of the 3 "bounce" dots that it could detect? 

Also it's a bit confusing that it is also nested inside the .form_error CSS rule?  But if we want to reuse componenents with styling tweaks there would always be an element of nesting unless we pass a context class down into the component?